src/ape_test/__init__.py

```diff
@@ -57,6 +57,8 @@ class Config(PluginConfig):
     Set to ``False`` to keep providers connected at the end of the test run.
     """
 
+    hd_path: str = "m/44'/60'/0'"
```

Comment:
This should be made into a global variable next to `DEFAULT_TEST_MNEMONIC` and `DEFAULT_NUMBER_OF_TEST_ACCOUNTS`

Then, `generate_dev_accounts` should make use of it for its `hd_path_format` default kwarg.

This way, it's all in 1 spot.